src/qibolab/instruments/qm/simulator.py

```diff
@@ -4,7 +4,7 @@
 from qm.QuantumMachinesManager import QuantumMachinesManager
 from qualang_tools.simulator_tools import create_simulator_controller_connections
 
-from qibolab.instruments.qm import QMOPX
+from qibolab.instruments.qm.driver import QMOPX
```

Comment:
I would start using relative imports, here you see it definitely makes sense (you only care about the relative relation between this file and `driver.py`).
```suggestion
from .driver import QMOPX
```

Reply:
Actually, I wanted to propose to use relative imports all over the place, and I collected a couple of references. But since this is common to all repos, I was thinking which would be a proper place to discuss.

Maybe we should just take into account changing scheme in Qibolab, and if we agree we can do the same for the other repos.